### PR TITLE
feat: TOOLS-2700 handle server removal of 'bins' info cmd

### PR DESCRIPTION
**Pull request by jdogmcsteezy**

### CODE REVIEW

**Review by dwelch-spike — COMMENTED** [review]
Just wanna make sure this handles mixed clusters, otherwise LGTM

File: lib/live_cluster/show_controller.py [hunk 1]
```diff
@@ -1178,10 +1180,24 @@ async def do_bins(self, line):
...
+        if any(
+            [
+                version.LooseVersion(build)
+                >= version.LooseVersion(constants.SERVER_INFO_BINS_REMOVAL_VERSION)
+                for build in builds.values()
+            ]
+        ):
+            self.logger.error(
+                f"Server version {constants.SERVER_INFO_BINS_REMOVAL_VERSION} removed namespace bin-name limits and statistics."
+            )
```
[the remark this answers is not in the document]
Comment (jdogmcsteezy) [reply lib/live_cluster/show_controller.py:1199]: Yes, if any are >= 7.0 then the error is displayed and the table will still be displayed for the nodes < 7.0

[the remark this answers is not in the document]
Comment (jdogmcsteezy) [reply lib/live_cluster/show_controller.py:1199]: Remember that the metadata_getter will fitler exceptions out so a node >= 7.0 will have an empty entry or no entry if it is missing

**Review by dwelch-spike — APPROVED** [review]
LGTM

**Pull request merged**